### PR TITLE
travis-deploy docker service

### PR DESCRIPTION
**Pull request by liam-lai**
#### Description
Let Travis deploy ecs docker service with aws secret.

#### QA URL

https://demo-roster.efcsydney.org/

#### Acceptance Criteria

- [ ] Travis can deploy this service on AWS

### CODE REVIEW

File: docker_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# create parameter
+if [ $1 = "qa" ]; then
```
Comment (kleinchang) [inline docker_deploy.sh]: Do we plan to use this script for qa & production deployment? In .travis.yml we run this script only on qa branch. And looks like we only append "-latest" in the name of QA image right?

File: docker_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+file='aws/ecs_service.json'
+stack='ecs-service'
+
+aws --region ap-southeast-2 cloudformation deploy \
+    --capabilities CAPABILITY_IAM \
+    --template-file $file \
+    --stack-name $stack \
+    --parameter-overrides \
+        Environment=$env \
+        Certificates=$cert \
```
Comment (kleinchang) [inline docker_deploy.sh]: Out of interest is this $cert defined in travis site? Can't see where it was defined here and .travis.yml.

Comment (liam-lai) [reply docker_deploy.sh]: Oh!! Nice find out. This is for ALB https certificate. I will remove it.

File: docker_deploy.sh (hunk 1 above)
Comment (liam-lai) [reply docker_deploy.sh]: 1. Plan to go for QA and Production. Need to sit with you on Production setting. 
2. ${BUILD_TAG:-latest} this means, if Build_TAG is not existed, then return "latest", kind of default value.

**Pull request merged**